config/htaccess.yaml

```diff
@@ -23,7 +23,7 @@
 ### Giza requires this file, even if no redirects
 ### More as a safety -- in case, we remove redirects during deploys
 #######################################################################
-### Redirects to put /v2.0/ in the URL                              ###
+### Redirects to put /current in the URL                                                                                    ###
```

Comment:
For htaccess, current works fine since redirects are only built on master branch.

For conf-sitemap, let's say we now have v2.0 branch and a master which corresponds to v2.1.
You don't want to build a sitemap for v2.0 using current. Also, I'm not sure when the current symlink is built so there might be an issue if the sitemap is built before the current -> master branch exists.